client/src/components/ElementEditor/Element.js

```diff
@@ -1,21 +1,29 @@
 import React, { PureComponent } from 'react';
 import { elementType } from 'types/elementType';
+import { inject } from 'lib/Injector';
 
 /**
  * The Element component used in the context of an ElementEditor shows the summary
  * of an element's details when used in the CMS, including ID, Title and Summary.
  */
 class Element extends PureComponent {
   render() {
-    const { element: { ID, Title, Summary } } = this.props;
+    const { element: { ID, Title, Summary, Type, Icon, Modified, Draft } } = this.props;
 
     if (!ID) {
       return null;
     }
 
     return (
       <div className="element-editor__element">
-        <p>#{ID}: {Title}</p>
+        <HeaderComponent
+          elementId={ID}
+          title={Title}
+          elementType={Type}
+          fontIcon={Icon}
+          modified={Modified}
+          draft={Draft}
```

Comment:
I'd like to make this more of a "versionedState" than individual props, but can be handled separately (see #276)